src/areaSearch/components/AreaSearchApplicationEdit.js

```diff
@@ -91,14 +96,56 @@ class AreaSearchApplicationEdit extends Component<Props, State> {
 
     initialize(getInitialAreaSearchEditForm(areaSearch));
   }
+  
+  handleFileAdded = (e: any) => {
```

Comment:
Can `e` be anything?

Reply:
Looking at the codebase, I think it is Event.